SwiftRuntimeLibrary/Enums.cs

```diff
@@ -114,5 +114,12 @@ public enum ProtocolDispatchStrategy {
 		ObjC = 0,
 		Swift = 1,
 	}
+
+	public enum SwiftProtocolConformanceTypeDescriptorKind {
+		DirectTypeDescriptor,
+		IndirectTypeDescriptor,
+		DirectObjCClassName,
+		IndirectObjCClass,
```

Comment:
```
DirectObjCClassName,
IndirectObjCClass,
```

one ends with `Name`, not the other ? are they the same _name_ thing [in]direct, or different things ?

Reply:
They are not the same. One is a relative pointer to a class name and the other is an indirect pointer to a class object.